mysite/_config/cwpadmin.yml

```diff
@@ -0,0 +1,7 @@
+---
+Name: cwpadmin
+---
+SilverStripe\Admin\LeftAndMain:
+  help_link: "https://www.cwp.govt.nz/user-help/"
+SilverStripe\Control\Email:
+  admin_email: 'no-reply@cwp.govt.nz'
```

Comment:
I feel that this could be in cwp/cwp-core, but at the same time we might want to let users override both of these values super easily, so it's probably OK here =D

Reply:
Yeah, I think it's a safe default that doubles as an example (i.e. _should_ be changed as part of development). We could of course move it, but then I think that would obscure how recipes work, in that it either 
* would never get applied - developers often use e.g. `mysite/_config/config.yml` so using that in cwp-core probably isn't ideal
* not be obvious that it is being applied, and/or that the developer should override it by essence that it comes form an external module (and that the general rule of thumb is to not edit things that come from external sources).

So... in the end, ... this is fine as is?

Reply:
> would never get applied - developers often use e.g. mysite/_config/config.yml so using that in cwp-core probably isn't ideal

Yeah - you'd rename it to something unique instead

> not be obvious that it is being applied, and/or that the developer should override it by essence that it comes form an external module (and that the general rule of thumb is to not edit things that come from external sources).

Yeah, but there's already plenty of that. I think if this were a mandated requirement that emails from from a *@cwp.govt.nz address, or that we keep the documentation links pointing at the CWP website, then it'd be fine to put it into cwp-core (note: still overridable by user config). A user might want to have their own set of CMS documentation for a client (in theory).

>So... in the end, ... this is fine as is?

👍